apps/test/interpreter/test-on-circle.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -x
+./node_modules/@code-dot-org/js-interpreter-tyrant/bin/run.js --threads 1 --run --diff --verbose
```

Comment:
Similar note here, but since `node_modules/.bin` isn't in scope you'd use
```bash
`npm bin`/run.js --threads 1 --run --diff --verbose
```
If we're dropping npm support, I suppose `` `yarn bin` `` is preferable - it works the same way.